challenges/ecosystem/02.md

```diff
@@ -42,30 +42,41 @@ Prettier is an opinionated code formatter. It enforces a consistent style by par
 
 ![Prettier format on save preview on VS Code](https://raw.githubusercontent.com/flexbox/react-native-workshop/main/challenges/ecosystem/format-on-save.png)
 
-### Setup ESLint rules for React Native project
 
-- [ ] Install ESLint as a dev dependency.
+### Setup ESLint in your expo project
 
 ```console
-npm install --dev eslint @typescript-eslint/eslint-plugin @typescript-eslint/parser
-# same as `yarn add --dev eslint @typescript-eslint/eslint-plugin @typescript-eslint/parser`
+npx expo lint
 ```
 
-- [ ] Create a `.eslintrc.js` file at the root of your project with the following content: (you can use `touch .eslintrc.js` from the terminal).
+You can lint your code manually by rerunning `npx expo lint` in your terminal.
 
-```javascript
-// .eslintrc.js
+- [ ] Setup ESLint in your project.
+- [ ] Run `npx expo lint` in your terminal.
+
+### Prettier
+
+For other packages managers
+```console
+npx expo install -- --save-dev prettier eslint-config-prettier eslint-plugin-prettier
```

Comment:
1.
by default `eslint-config-expo` is using the prettier plugin https://github.com/expo/expo/blob/main/packages/eslint-config-expo/.eslintrc.js

So i don't think we need it, but I may be wrong 😅 

2.
```
npx expo install
```
is only used for `expo-*` or `react-native-*` packages, replace with `npm` :pray: